server-side-scanners/boostsecurityio/sbom-sca/rules.yaml

```diff
@@ -15,3 +15,13 @@ rules:
     group: top10-vulnerable-components
     pretty_name: Dependency with known malicious behaviour
     ref: https://github.com/ossf/malicious-packages/tree/main/osv/malicious
+  forbidden-license:
```

Comment:
I thought Stephan wanted to import from the license scanner

Reply:
Good point. I did the import from other scanners feature too 🤦

Reply:
mmh, yeah I don't think import works for server-side scanners... I don't remember it this is a bug or a restriction that we wanted. I'll check the action quickly. If there's no easy solution, I can create a rules-realm for it or just copy the rules for now.

Reply:
I was able to update the action to allow imports from another server-side scanner: https://github.com/boostsecurityio/scanner-registry-action/pull/48